src/microbe_stage/systems/StrainSystem.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace Systems
+{
+    using Components;
+    using DefaultEcs;
+    using DefaultEcs.System;
+    using DefaultEcs.Threading;
+    using Godot;
+    using World = DefaultEcs.World;
+
+    [With(typeof(CompoundStorage))]
+    [With(typeof(OrganelleContainer))]
+    [With(typeof(MicrobeControl))]
```

Comment:
As far as I can tell these two components are just read from and as such should have the reads from component attributes here to mark that to make multithreading more effective related to this system.